### DISCUSSION TITLE
Developing an Advanced Image Recognition Model for Bird Nest Detection in UAV Imagery (GSOC 2024)

### DISCUSSION THREAD

**Discussion by Om-Doiphode** [weecology/DeepForest#618] [General]
I am Om Doiphode, a third-year undergraduate student in Computer Engineering at VJTI Mumbai. I have experience in Deep learning particularly in the field of Computer Vision. I am an ML intern at an AI startup in Sunnyvale, California. I am interested in the project 
[Developing an Advanced Image Recognition Model for Bird Nest Detection in UAV Imagery](https://github.com/weecology/retriever/wiki/GSoC-2024-Project-Ideas#proposal-developing-an-advanced-image-recognition-model-for-bird-nest-detection-in-uav-imagery). My approach was to build a one-shot detector for the project like YOLO or SSD. Can you please guide me further on this project? I have set up Deepforest successfully in my system and merged 3 pull requests in the Deepforest repository.

**Comment by henrykironde** [comment]
@Om-Doiphode,I suggest reviewing the documents again and beginning with a sample tree model. Once you feel confident with that let us know.

**Reply by Om-Doiphode** [reply]
What did you mean by `documents`? Did you mean the DeepForest's documentation? Can you please elaborate?

**Comment by henrykironde** [comment]
Yes, I mean the Documentation.

**Reply by Om-Doiphode** [reply]
I have read the documentation and tested the examples on my system for the tree model. Can you please tell me on what to do further?

**Reply by henrykironde** [reply]
Search for existing issues and work on resolving them. This will deepen your comprehension of the codebase, facilitating a transition from tree modeling to bird modeling.
